### PR TITLE
Better documentation for metadata

### PR DESCRIPTION
**Pull request by ger-benjamin**
Fix: #2870 

Ready for a review, please, say me if it's understandable (and correct spelling).

(look also: https://github.com/camptocamp/c2cgeoportal/issues/3085)

### CODE REVIEW

File: contribs/gmf/externs/gmf-themes.js [hunk 1]
```diff
@@ -415,139 +421,200 @@ gmfThemes.GmfMetaData.prototype.copyable;
...
+ * For WMTS layers.
  * @type {string|undefined}
  */
 gmfThemes.GmfMetaData.prototype.ogcServer;
 
 
 /**
- * Layer opacity, default is 1.0
+ * Layer opacity.
+ * Default to 1.0 (full visible, 0 means invisible)
```
Comment (gberaudo) [inline contribs/gmf/externs/gmf-themes.js]: fully

File: contribs/gmf/externs/gmf-themes.js [hunk 2]
```diff
@@ -405,8 +401,18 @@ gmfThemes.GmfMetaData = function() {};
 
 
 /**
- * Whether the data of of this layer node can be queried in order to be copied
- * in an other layer.
+ * Names of layers on which the geometry can be copied to (in the edition).
```
Comment (gberaudo) [inline contribs/gmf/externs/gmf-themes.js]: in edition mode / edit mode

**Comment by adube** [comment]
@ger-benjamin 

https://github.com/camptocamp/ngeo/blob/2.2/options/ngeox.js#L204-L209

```
/**
 * Whether the geometry from this data source can be copied to other data
 * sources or not. Defaults to `false`.
 * @type {boolean|undefined}
 */
ngeox.DataSourceOptions.prototype.copyable;
```

Please, let me know if that's clear enough.

**Comment by ger-benjamin** [comment]
Thanks  @adube, @gberaudo and @llienher :-)

**Pull request merged**